code/__DEFINES/vehicles.dm

```diff
@@ -6,9 +6,16 @@
 
 //Ridden vehicle flags
 
-#define REQUIRES_ARMS   (1<<0)    //Does our vehicle require hands to drive?
-#define REQUIRES_LEGS   (1<<1)    //Does our vehicle require legs to drive?
-#define UNBUCKLE_DISABLED_RIDER (1<<2)   //If our rider is disabled, does he fall off?
+/// Does our vehicle require arms to operate? Also used for piggybacking on humans to reserve arms on the rider
+#define RIDER_NEEDS_ARMS   (1<<0)
```

Comment:
Maybe clarify this is a check for specifically two or more arms?
Something like `RIDER_NEEDS_MULTIPLE_ARMS` vs `RIDER_NEEDS_ONE_ARM`?